docs/overview/what_is__-__main_areas/con-serverless-function-processing-stages.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Function processing
```

Comment:
I'm still not sure what's the relationship between what is called "conditions", "phases", and "stages". Are those synonyms? I would like to change the title of this file to "Phases of Functions" or something like that, but first I need to understand what those words mean exactly.